.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -27,3 +28,27 @@ I have:
 
 [1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a
 [2]: https://www.notion.so/passcultureapp/cb45383351b44723a6f2d9e1481ad6bb?v=10fe47258701423985aa7d25bb04cfee&pvs=4
+
+## Best Practices
+
+<details>
+  <summary>Click to expand</summary>
+
+- Don't use type assertions with `as` (type assertions are removed at compile-time, there is no runtime checking associated with a type assertion. There won’t be an exception or `null` generated if the type assertion is wrong).
+- Don't opt out of type checking with `any` (when you want to accept anything because you will be blindly passing it through without interacting with it, you can use `unknown`).
+- Don't use non-null assertion operators (just like other type assertions, this doesn’t change the runtime behavior of your code, so it’s important to only use `!` when you know that the value can’t be `null` or `undefined`).
+- When creating a new component, start by creating it in StoryBook.
+- When creating a new logic, start by writing tests (TDD).
+- Avoid (or remove) all `@ts-expect-error` and `@eslint-disable`. If not avoidable, specify why with a comment.
+- Remove all warnings, and "normal errors".
+- Write tests for existing code if there were none.
+- Use `gap` instead of `<Spacer.Column />` or `<Spacer.Row />` when it is more optimal.
+
+Test specific:
+
+- In tests, mock external calls, but don't mock internal parts of our code.
+- When you see a local variable that is overwitten in every test, mock it.
+- In tests, prefer `user` to `fireEvent`.
+- Replace `await waitFor(/* ... */)` or `await act(async () => {})` by `await screen.findByTruc()`.
```

Comment:
juste au dessus, c'est marqué "Test specific", je pense qu'on peut supprimer "In tests,"

```suggestion
Test specific:

- Mmock external calls, but don't mock internal parts of our code.
- When you see a local variable that is overwitten in every test, mock it.
- Prefer `user` to `fireEvent`.
- Replace `await waitFor(/* ... */)` or `await act(async () => {})` by `await screen.findByTruc()`.
```